functorch/test/test_ops.py

```diff
@@ -348,13 +348,12 @@ def wrapped_fn(*args, **kwargs):
         # Composite ops that do bad things. Need to be fixed in PyTorch core.
         # RuntimeError: Cannot access data pointer of Tensor that doesn't have storage
         xfail('tensor_split'),
+        xfail('nn.functional.rrelu'),
```

Comment:
nit: could we at minimum keep the comment from the previous version? It just helps us remember what the error looks like

Reply:
Whoops